ova/.vagrant/rgloader/loader.rb

```diff
@@ -0,0 +1,9 @@
+# This file loads the proper rgloader/loader.rb file that comes packaged
```

Comment:
This file isn't necessary. You can create a `.gitignore` file to avoid committing anything from `.vagrant`.

Reply:
Done in https://github.com/wazuh/wazuh-packages/pull/223/commits/2629f18c2d161e7cfe70b71d39522548b5459f55